rasterio/windows.py

```diff
@@ -350,6 +350,42 @@ def window_index(window):
     return tuple(slice(*w) for w in window)
 
 
+def round_window_to_full_tiles(window=None, block_shapes=None):
```

Comment:
Can we make `window` and `block_shapes` positional arguments since they're both required.